ADR/Analysis/Stability/FlightStability/tests/test_FlightStability.py

```diff
@@ -0,0 +1,89 @@
+from ADR.Analysis.Stability.FlightStability.FlightStability import FlightStability
+from ADR.parameters import get_plane_parameters
+from ADR.Core.data_manipulation import dict_to_dataframe
+from ADR.Components.Plane import Plane
+import numpy.testing as npt
+import pandas as pd
+import pytest
+
+
+@pytest.fixture
+def plane():
+    plane_parameters = get_plane_parameters()
+    plane = Plane(plane_parameters)
+    return plane
+
+
+@pytest.fixture
+def flightStabilyty_analysis(plane):
+    flightStabilyty_analysis = FlightStability(plane)
+    return flightStabilyty_analysis
+
+
+def test_CM_plane_CG(flightStabilyty_analysis):
+    dict_of_CM_ALPHA_dataframes = flightStabilyty_analysis.CM_plane_CG(0.10)
+    CM_ALPHA_dataframe_for_hs_at_theta_12 = dict_of_CM_ALPHA_dataframes[12.0]
+    CM_ALPHA_dataframe_for_hs_at_theta_10_negative = dict_of_CM_ALPHA_dataframes[-10.0]
+    CM_ALPHA_dataframe_for_hs_at_theta_3_negative = dict_of_CM_ALPHA_dataframes[-3.0]
+    CM_ALPHA_dataframe_for_hs_at_theta_8 = dict_of_CM_ALPHA_dataframes[8.0]
+    CM_at_hs_theta_12_alpha_7 = CM_ALPHA_dataframe_for_hs_at_theta_12.at[7.0, 'CM']
+    CM_at_hs_theta_10_negative_alpha_ = CM_ALPHA_dataframe_for_hs_at_theta_10_negative.at[-8.0, 'CM']
+    CM_at_hs_theta_3_negative_alpha_ = CM_ALPHA_dataframe_for_hs_at_theta_3_negative.at[
+        0.0, 'CM']
+    CM_at_hs_theta_8_alpha_ = CM_ALPHA_dataframe_for_hs_at_theta_8.at[-4.0, 'CM']
+    npt.assert_almost_equal(CM_at_hs_theta_12_alpha_7, -0.264243, decimal=6)
+    npt.assert_almost_equal(
+        CM_at_hs_theta_10_negative_alpha_, -0.458495, decimal=6)
```

Comment:
[Travis is complaining](https://travis-ci.com/github/CeuAzul/ADR/builds/177676203) that this value should be 0.45893773522228737. Precision problem. 

You can use `decimal=3` and it should be fine. We don't need precision to the 6th digit.